dora-metrics/pelorus
# fix: Add file extensions to all files

Signed-off-by: Mateus Oliveira <msouzaol@redhat.com>

## Describe the behavior changes introduced in this PR

Add file extensions to all files. This enable the quality tools to check all of them.

We can add them to makefile or console scripts to avoid calling them with the extension.

## Linked Issues

resolves #684
depends on #944 

## Testing Instructions

I have run `find . -type f ! -name "?*.*" | grep -v "venv/\|git/\|.pytest_cache/\|htmlcov/"` to check for files without extension (there is still one that I think should have, that is `./pelorus-operator/PROJECT`, but it is generated by script, so we need to change script to change it).

To check if I have changed all needed places of invocation of the file, I have run `grep -Iinr 'name_of_the_file' . --exclude-dir={.venv,htmlcov,.pytest_cache,.mypy_cache}`.
